packages/dds/tree/src/test/simple-tree/mapTree.spec.ts

```diff
@@ -1498,3 +1499,31 @@ describe("toMapTree", () => {
 		});
 	});
 });
+
+describe("mapTrees", () => {
+	it("can be deep copied", () => {
```

Comment:
Nit: When testing a specific function, I generally just name the test the same as the function. Makes it easier to discover IMO.

Reply:
Yea good idea